CONTRIBUTING.md

```diff
@@ -1,58 +1,41 @@
 # Contributing to Mypy
 
-Welcome!  Mypy is a community project that aims to work for a wide
-range of Python users and Python codebases.  If you're trying mypy on
+welcome! basedmypy is a community project that aims to work for a wide
+range of Python users and Python codebases. If you're trying basedmypy on
 your Python code, your experience and what you can contribute are
-important to the project's success.
-
-## Code of Conduct
-
-Everyone participating in the Mypy community, and in particular in our
-issue tracker, pull requests, and chat, is expected to treat
-other people with respect and more generally to follow the guidelines
-articulated in the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
+important to the project's success
 
 ## Getting started with development
 
 ### Setup
 
 #### (1) Fork the mypy repository
 
-Within Github, navigate to <https://github.com/python/mypy> and fork the repository.
+Within Github, navigate to <https://github.com/KotlinIsland/basedmypy> and fork the repository.
 
 #### (2) Clone the mypy repository and enter into it
 
-```bash
-git clone git@github.com:<your_username>/mypy.git
-cd mypy
-```
+clone the project using your IDE
 
-#### (3) Create then activate a virtual environment
-
-```bash
-python3 -m venv venv
-source venv/bin/activate
+if you don't use an IDE, you can also use the terminal (not recommended, the terminal is better left in the 80's):
+```shell
+git clone git@github.com:<your_username>/basedmypy.git
```

Comment:
is this using the ssh method (never works) instead of the https method?

Reply:
trueeee